doc/classes/Particles2D.xml

```diff
@@ -39,8 +41,10 @@
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			This setting can be used to set the particle system to render at a fixed FPS. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+			This can be used to turn Fractional Delta on or off, defaults to [code]true[/code]. Setting fractional delta to [code]true[/code] results in sub delta calculation having a smoother particles display effect 
```

Comment:
As above. "Fractional Delta" shouldn't be capitalized. Boolean properties have a prescribed description per the docs guidelines:

If [code]true[/code], ... Default value: [code]true[/code].